pandas/tests/io/test_excel.py

```diff
@@ -105,6 +105,7 @@ def get_exceldf(self, basename, ext, *args, **kwds):
 class ReadingTestsBase(SharedItems):
     # This is based on ExcelWriterBase
 
+    @td.skip_if_no('xlrd', '1.0.1')  # GH-22682
```

Comment:
Could we alternately just bump the minimum version of xlrd? 1.0.0 is over 2 years old at this point. Only 1.1.0 has been released since then but even that is over a year old now

Reply:
Thought about it, though this seemed easier for now. I don't have a strong preference.